src/tools/annotation/bidirectionalTool/renderToolData.js

```diff
@@ -55,7 +54,7 @@ export default function(evt) {
       continue;
     }
 
-    color = data.active ? activeColor : toolColors.getToolColor();
+    color = toolColors.getColorIfActive(data);
```

Comment:
I added automated tests for the tools, so it should be possible, and fairly easy to add test coverage there if you want.